WcaOnRails/app/javascript/wca/currenciesData.js.erb

```diff
@@ -10,5 +9,6 @@ window.wca = window.wca || {};
                     :subunit_to_unit => v[:subunit_to_unit]}]
   end]
 %>
-wca._currenciesInfo =
-<%= raw(JSON.pretty_generate(currencies_info_hash, indent: '', object_nl: '', array_nl: '', space: '', space_after: '')) %>;
+export default {
+  byIso: <%= JSON.pretty_generate(currencies_info_hash, indent: '', object_nl: '', array_nl: '', space: '', space_after: '') %>,
+};
```

Comment:
Two comments here:
1. Is there any reason of using this `pretty_generate` method with an ugly options list? If not, then let's follow what @jfly did https://github.com/thewca/worldcubeassociation.org/blob/2c1b14318a5176668541a309972eefd0ddc52f08/WcaOnRails/app/javascript/wca/formats.js.erb#L4 So `currencies_info_hash.to_json.html_safe` in this case.
2. I'd suggest camelifying the keys as we already do, see [formats](https://github.com/thewca/worldcubeassociation.org/blob/2c1b14318a5176668541a309972eefd0ddc52f08/WcaOnRails/app/javascript/wca/formats.js.erb) for an example. (Note: then you need to fix your code to use the camel case names)